Packs/StellarCyber/Integrations/StellarCyber/StellarCyber.py

```diff
@@ -0,0 +1,441 @@
+    demisto.debug(f"get_remote_data_command: {args}")
+    parsed_args = GetRemoteDataArgs(args)
+    demisto.debug(f"parsed_args: {parsed_args}")
+    try:
+        remote_incident_id = parsed_args.remote_incident_id
+        incident = client.get_incident(remote_incident_id)
+        if len(incident):
+            incident_id = incident["_id"]
+            incident["case_url"] = f"https://{client.dp_host}/cases/case-detail/{incident_id}"
+            incident_ticket_id = str(incident["ticket_id"])
+            demisto.info(f"Retrieved case: {str(incident['ticket_id'])}")
+            if len(incident['metadata'].get('name_auto', [])):
+                incident_name = incident['metadata']['name_auto'][0]
+            else:
+                incident_name = incident['name']
+            incident["name"] = incident_name
+            incident_summary = client.get_incident_summary(incident_id)
+            incident["summary"] = incident_summary
+            event_ids = incident.get("event_ids", None)
+            security_event_cnt = len(event_ids)
+            demisto.info(
+                "Pulling security event info for case: [{}] [ticket id: {}] [event_cnt: [{}]".format(
+                    incident_id, incident_ticket_id, security_event_cnt
+                )
+            )
+            incident["security_alerts"] = []
+            for event in event_ids:
+                incident["security_alerts"].append(client.get_alert(alert_id=event["_id"], alert_index=event["_index"]))
+            incident["severity"] = get_xsoar_severity(incident["priority"])
+            return GetRemoteDataResponse(mirrored_object=incident, entries=[])
+        else:
+            return_error(f"Failed to retrieve case: {str(incident['ticket_id'])}")
+    except Exception as e:
+        if "Rate limit exceeded" in str(e):
+            return_error("API rate limit")
+
+
+def get_modified_remote_data_command(client: Client, args) -> GetModifiedRemoteDataResponse | str | None:
+    demisto.debug(f"get_modified_remote_data_command: {args}")
+    try:
+        remote_args = GetModifiedRemoteDataArgs(args)
+        last_update = remote_args.last_update
+        demisto.debug(f"last_update: {last_update}")
+        last_update_utc = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})  # type: ignore
+        demisto.debug(f"last_update_utc: {last_update_utc}")
+        assert last_update_utc is not None
+        last_run_ts = int((last_update_utc - timedelta(minutes=1)).timestamp() * 1000)
+        modified_incidents = client.get_updated_incidents(last_run=last_run_ts)
+        if len(modified_incidents):
+            modified_incident_ids = [
+                f"{str(i['ticket_id'])}:{i['cust_id']}"
+                for i in modified_incidents
+            ]
+        else:
+            modified_incident_ids = []
+        return GetModifiedRemoteDataResponse(modified_incident_ids)
+    except Exception as e:
+        return_error("skip update")
```

Comment:
search commands should return a readable output when not found, not errors.
Please return a human readable message and no Context data